{"uid":"baa855f4147fc4a8","name":"test_log_rotation","fullName":"tests.integration.test_log_rotation#test_log_rotation","historyId":"cee6faeba82de98eaf891ec9dd9520e1","time":{"start":1731897182693,"stop":1731898085922,"duration":903229},"description":"Test log rotation.","descriptionHtml":"<p>Test log rotation.</p>\n","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1788, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/tests/integration/test_log_rotation.py\", line 81, in test_log_rotation\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1040, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 131, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1731897182400,"stop":1731897182401,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1731897182401,"stop":1731897182692,"duration":291},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1731897182401,"stop":1731897182401,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1731897182692,"stop":1731897182692,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test log rotation.","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1788, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/tests/integration/test_log_rotation.py\", line 81, in test_log_rotation\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1040, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 131, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","steps":[],"attachments":[{"uid":"e6153e860f32202b","name":"log","source":"e6153e860f32202b.txt","type":"text/plain","size":4204}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1731898085983,"stop":1731898085983,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1731898085984,"stop":1731898092751,"duration":6767},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1731898092751,"stop":1731898092751,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1731898092751,"stop":1731898092752,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1731898092752,"stop":1731898092752,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_log_rotation"},{"name":"host","value":"fv-az1771-307"},{"name":"thread","value":"12500-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_log_rotation"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"2fe304c9ee2d52a","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":99,"skipped":0,"passed":63,"unknown":10,"total":172},"items":[{"uid":"e2097461ed155b14","reportUrl":"../350//#testresult/e2097461ed155b14","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731810867211,"stop":1731811770615,"duration":903404}},{"uid":"f23dac9d51953776","reportUrl":"../349//#testresult/f23dac9d51953776","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731724067021,"stop":1731724969276,"duration":902255}},{"uid":"e208935b803b2983","reportUrl":"../347//#testresult/e208935b803b2983","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731637819666,"stop":1731638722281,"duration":902615}},{"uid":"4bef0dd71f8a7795","reportUrl":"../346//#testresult/4bef0dd71f8a7795","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731551087336,"stop":1731551990154,"duration":902818}},{"uid":"d41905a1072afe49","reportUrl":"../345//#testresult/d41905a1072afe49","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731464602818,"stop":1731465505329,"duration":902511}},{"uid":"75494a3227bf3543","reportUrl":"../344//#testresult/75494a3227bf3543","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731378220088,"stop":1731379123383,"duration":903295}},{"uid":"1ac6a96c6adaa256","reportUrl":"../343//#testresult/1ac6a96c6adaa256","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731292000660,"stop":1731292903192,"duration":902532}},{"uid":"e6168ecd482298f8","reportUrl":"../342//#testresult/e6168ecd482298f8","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731205753613,"stop":1731206656049,"duration":902436}},{"uid":"3a1006e017424f19","reportUrl":"../341//#testresult/3a1006e017424f19","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731119148968,"stop":1731120051305,"duration":902337}},{"uid":"d17cd90586d088a1","reportUrl":"../340//#testresult/d17cd90586d088a1","status":"unknown","time":{}},{"uid":"1bcdabe383d1784a","reportUrl":"../338//#testresult/1bcdabe383d1784a","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1730946269418,"stop":1730947171700,"duration":902282}},{"uid":"c603f472c2010da3","reportUrl":"../337//#testresult/c603f472c2010da3","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1730859826102,"stop":1730860728427,"duration":902325}},{"uid":"1be9c680d8ceb60c","reportUrl":"../336//#testresult/1be9c680d8ceb60c","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1730773504730,"stop":1730774407865,"duration":903135}},{"uid":"df3187da97940e08","reportUrl":"../334//#testresult/df3187da97940e08","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1730687356035,"stop":1730688258453,"duration":902418}},{"uid":"afd37b1889c08b68","reportUrl":"../333//#testresult/afd37b1889c08b68","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1730601129779,"stop":1730602032234,"duration":902455}},{"uid":"3eeaa5ad0440e542","reportUrl":"../332//#testresult/3eeaa5ad0440e542","status":"unknown","time":{}},{"uid":"d40701833bc8e52f","reportUrl":"../331//#testresult/d40701833bc8e52f","status":"unknown","time":{}},{"uid":"138705fbe2afd44","reportUrl":"../329//#testresult/138705fbe2afd44","status":"unknown","time":{}},{"uid":"6df44b79ca65779f","reportUrl":"../326//#testresult/6df44b79ca65779f","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1730168958937,"stop":1730169860941,"duration":902004}},{"uid":"788b2d6b9c3f63e1","reportUrl":"../325//#testresult/788b2d6b9c3f63e1","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1730082697271,"stop":1730083600187,"duration":902916}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"baa855f4147fc4a8.json","parameterValues":[]}